pkg/system/phase2_creating.go

```diff
@@ -353,3 +418,88 @@ func (r *Reconciler) SetDesiredAgentProfile(profileString string) string {
 	util.Panic(err)
 	return string(profileBytes)
 }
+
+// fixPvs make changes related to pvs if an upgrade process is needed
+func (r *Reconciler) fixPvs() {
```

Comment:
I suggest a more specific name
```suggestion
func (r *Reconciler) UpgradeRenameDBVolume() {
```